src/hooks/Options.py

```diff
@@ -45,12 +44,16 @@ def before_options_defined(options: dict) -> dict:
     return options
 
 # This is called after any manual options are defined, in case you want to see what options are defined or want to modify the defined options
-def after_options_defined(options: dict) -> dict:
-    # The generated goal option will not keep your defined values or documentation string unless you add them to option.json or you add them here:
-    # To automatically convert your own Goal class to alias of the generated goal uncomment the lines below and replace 'Goal' with your own option of type Choice
+def after_options_defined(options: PerGameCommonOptions):
+    # To access a modifiable version of options check the dict in options.__annotations__
+    # For example if you want to change DLC_enabled's display name you would do:
+    # options.__annotations__["DLC_enabled"].display_name = "New Display Name"
+
+    # The generated goal option will not keep your defined values or documentation string you'll need to add them here:
+    # To automatically convert your own goal to alias of the generated goal uncomment the lines below and replace 'Goal' with your own option of type Choice
```

Comment:
We don't want to be encouraging people to replace Goal entirely.

Reply:
It's not replacing its converting old goal values and alias to aliases to generated goal

Reply:
Yeah, the examples in the comments in this might be confusing for some people. Would be good to have a clear separation between option definitions (which are their own custom ones that they make) and option modifications, which follow the same format for modifications after the fact.

So the goal example should just be `options.__annotations__.get("goal").whatever = whatever` for any of the props, instead of constructing a custom Goal option just to siphon off some properties.